src/tests/Loader/classloader/StaticVirtualMethods/Regression/GitHub_74132.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+// This regression test tracks the issue where GetInterfaceMap returns incorrect
```

Comment:
This is regression test for a BCL API behavior. It would be more appropriate to add it under src\libraries, similar to https://github.com/dotnet/runtime/pull/90518 - if this case is not covered by the tests being added already.

Reply:
OK, thanks for pointing that out, in such case I'll abandon this PR and I'll follow up on the #90518 PR thread.